docs/API.md

```diff
@@ -195,29 +203,41 @@ The object has three keys, `"module"`, `"message"` and `"level"`.
 
 Sometimes additional keys are present.
 
-* `"ns"`: a *domain name*. The name server used by the *test module*.
-
->
-> TODO: Can other extra keys in addition to `"ns"` occur here? Can something be said
-> about when each extra key is present?
->
+* `"ns"`: a *domain name*. The name server used by the *test module*. 
+This key is added when the module name is `"NAMESERVER"`.
 
 
 ### Timestamp
 
 Basic data type: string
 
+Default database timestamp format: "Y-M-D H:M:S.ms"
+Example: "2017-12-18 07:56:17.156939"
```

Comment:
I think the format should prescribed. Not just default format.

Reply:
@mtoma says Timestamp is only used in responses. This PR is about request parameters. I see no need to get this right before merging.

Reply:
I can accept that. In an upcoming update of the API we should get it right.